packages/snap-store-mobx/src/Autocomplete/Stores/AutocompleteQueryStore.ts

```diff
@@ -1,12 +1,17 @@
 import { observable, makeObservable } from 'mobx';
 import type { UrlManager } from '@searchspring/snap-url-manager';
-import type { SearchResponseModelSearch, AutocompleteResponseModelAllOfAutocomplete } from '@searchspring/snapi-types';
+import type {
+	SearchResponseModelSearch,
+	AutocompleteResponseModelAllOfAutocomplete,
+	SearchResponseModelSearchMatchTypeEnum,
+} from '@searchspring/snapi-types';
 import type { AutocompleteStoreConfig, StoreServices } from '../../types';
 
 export class AutocompleteQueryStore {
 	public query?: Query;
 	public originalQuery?: Query;
 	public correctedQuery?: Query;
+	public matchType?: SearchResponseModelSearchMatchTypeEnum;
```

Comment:
Seems we may need to alter SnapiOAS so that it includes `primary` as a possible type?


https://github.com/searchspring/snapi-types/blob/v0.1.38/models/SearchResponseModelSearch.d.ts#L48-L50